tox.ini

```
@@ -22,10 +22,12 @@ commands =
 [testenv:lint]
 basepython = python3
 deps =
+  black
```

We should pin the black version here, new releases of black can change the behavior which will cause CI failures and also a potentially a ping pong issue with local environments if there is a mismatch between versions. So we want to make sure everyone is running the same version of black.

```suggestion
  black==21.5b0
```